README.md

```diff
@@ -299,13 +303,46 @@ laptop asset should be converted to a folder, and the license moved into it.
   Pass `git-command-args` as arguments to `git`, using the Onyo repository as
   the git repository. This is most valuable when used in conjunction with
   `ONYO_REPOSITORY_DIR`.
-
+- `onyo new [--non-interactive, -I] directory`:
+
+  Creates a new `asset` in `directory`. The command opens a dialog that asks for
+  the field names defined by the asset name scheme, and after creation opens the
+  new `asset` file with the editor.
+  After the editing is done, the new file will be checked for the validity of
+  it's YAML syntax.
+
+  - `--non-interactive` : Suppress opening of editor after file creation.
+- `onyo history [--non-interactive, -I] asset | directory`:
+
+  Show the history of a `directory` or `asset` file. The tools to show the
+  history are defined in `.onyo/config`.
+  By default, to show the history in interactive mode, the command uses
+  `tig --follow asset | directory`, and for the non-interactive mode it calls
+  `git --no-pager log --follow asset | directory`. The default tools can be
+  changed with `onyo config`.
+
+  - `--non-interactive` : Use the non-interactive tool to show the history of a
+    `asset` or `directory`.
+- `onyo config variable value`:
+
+  Set a `variable` to `value` in the `.onyo/config` file. This command can for
+  example change the default tool for the interactive mode of `onyo history`
+  with `onyo config history.interactive "git log --follow"`.
+- `onyo fsck`:
+
+  Checks if an `onyo repository` is given, which needs to be a valid git
```

Comment:
I would like an opening more like: "Runs a comprehensive suite of checks to verify the integrity of the repository and contents of an onyo repository. Including: <list>"

Reply:
Yes, sounds good. I started with that as a quick introduction sentence, before going into details.